repository/src/main/java/org/apache/atlas/repository/audit/ESBasedAuditRepository.java

```diff
@@ -243,6 +254,32 @@ private EntityAuditSearchResult getResultFromResponse(String responseString) thr
                 eventKey = event.getEntityId() + ":" + event.getTimestamp();
             }
 
+            Map<String, Object> detail = event.getDetail();
+            if (detail != null && detail.containsKey("attributes")) {
+                Map<String, Object> attributes = (Map<String, Object>) detail.get("attributes");
+
+                for (Map.Entry<String, Object> entry: attributes.entrySet()) {
+                    if (linkedAttributes.contains(entry.getKey())) {
+                        List<String> guids = (List<String>) entry.getValue();
+
+                        if (guids != null && !guids.isEmpty()){
+                            for (String guid: guids){
+                                if(!existingLinkedEntities.containsKey(guid)){
+                                    try {
+                                        AtlasEntityHeader entityHeader = fetchAtlasEntityHeader(guid);
+                                        if (entityHeader != null) {
+                                            existingLinkedEntities.put(guid, entityHeader);
+                                        }
+                                    } catch (AtlasBaseException e) {
```

Comment:
Since you have cached this exception here, let us add LOG.error explaining exception caught at extracting linked entities for events with a specific `eventKey` printed in error message